lib/indexer/index_documents.rb

```diff
@@ -82,7 +75,7 @@ def update_document_in_search_index(document, links)
     end
 
     def should_index_document?(content_item)
-      MIGRATED_TAGGING_APPS.include?(content_item.fetch("publishing_app")) &&
+      !NON_MIGRATED_APPS.include?(content_item.fetch("publishing_app")) &&
```

Comment:
This is now pretty hard to understand probably. Perhaps we could flip the naming (`should_skip_document?`) and simplify the conditional?